### ISSUE TITLE
[Bug] Uncaught TypeError: Cannot read properties of undefined (reading '__ec_inner_22')

### ISSUE THREAD

**Issue by nhhaidee** [apache/echarts#17657]
### Version

5.3.3

### Steps to Reproduce

Hi,

My application has problem with error `Uncaught TypeError: Cannot read properties of undefined (reading '__ec_inner_22')` and I opened this issue here just want to consult with dev team what error mean? Errors thrown from below Echarts code

```
} catch (e) {
      this[IN_MAIN_PROCESS_KEY] = false;
      throw e;
    }
```

My application is complicated so I am still trying to narrow down problem and will provide minimal test to reproduce problem later. I hope for now that dev team can help me to figure out what is problem

```
 ECharts.prototype.resize = function (opts) {
    if (this[IN_MAIN_PROCESS_KEY]) {
      if (true) {
        (0,_util_log_js__WEBPACK_IMPORTED_MODULE_11__.error)('`resize` should not be called during main process.');
      }

      return;
    }

    if (this._disposed) {
      disposedWarning(this.id);
      return;
    }

    this._zr.resize(opts);

    var ecModel = this._model; // Resize loading effect

    this._loadingFX && this._loadingFX.resize();

    if (!ecModel) {
      return;
    }

    var needPrepare = ecModel.resetOption('media');
    var silent = opts && opts.silent; // There is some real cases that:
    // chart.setOption(option, { lazyUpdate: true });
    // chart.resize();

    if (this[PENDING_UPDATE]) {
      if (silent == null) {
        silent = this[PENDING_UPDATE].silent;
      }

      needPrepare = true;
      this[PENDING_UPDATE] = null;
    }

    this[IN_MAIN_PROCESS_KEY] = true;

    try {
      needPrepare && prepare(this);
      updateMethods.update.call(this, {
        type: 'resize',
        animation: (0,zrender_lib_core_util_js__WEBPACK_IMPORTED_MODULE_3__.extend)({
          // Disable animation
          duration: 0
        }, opts && opts.animation)
      });
    } catch (e) {
      this[IN_MAIN_PROCESS_KEY] = false;
      throw e;
    }

    this[IN_MAIN_PROCESS_KEY] = false;
    flushPendingActions.call(this, silent);
    triggerUpdatedEvent.call(this, silent);
  };

```

### Current Behavior

Error:

```
Uncaught TypeError: Cannot read properties of undefined (reading '__ec_inner_20')
    at eval (model.js:603:19)
    at DataDiffer.eval [as _remove] (CustomView.js:198:169)
    at DataDiffer._executeOneToOne (DataDiffer.js:161:30)
    at DataDiffer.execute (DataDiffer.js:128:75)
    at CustomChartView.render (CustomView.js:202:8)
    at Task.progress (Chart.js:247:20)
    at Task._doProgress (task.js:193:10)
    at Task.perform (task.js:159:16)
    at eval (echarts.js:1893:24)
    at GlobalModel.eval (Global.js:668:10)
```

### Expected Behavior

Should be no error

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

### Any additional comments?

The problem does not happen with Echarts v5.2.2 but happen since 5.3.0

**Comment by plainheart** [comment]
It looks similar to the issues fixed by #17308. Please try `v5.4.0-rc.1` and check if it works.

**Comment by nhhaidee** [comment]
The problem seems to be fixed with v5.4.0-rc.1!

**Comment by plainheart** [comment]
Thanks for your check. The v5.4.0 is under the vote and near release.
